### DISCUSSION TITLE
zh/start/join

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#730] [Announcements]
# zh/start/join

介绍： 欢迎来到我们的社区！要加入 Discord 服务器并开始使用 Korepi，请按照以下步骤操作。 1.创建 Discord 帐户。 如果您还没有 Discord 帐户，请首先创建一个新帐户。访问 官方 Discord 网站 (https://discord.com/) 并按照注册说明进行操作。 2.下载和安装 Discord。 创建帐户后，为您...

https://korepi.com/zh/start/join.html

**Comment by genshinyubao** [comment]
Please edit the invitation link again, it is no longer valid. Come on
